salesagility/SuiteCRM
# Fix #3088 - Send correct password reset email

## Description
Fix for #3088  - When logged in as the Admin user and you go to reset a user's password, the password sent is always the admin user's password.

## Motivation and Context
Bug raised by a user. Broken functionality due to caching wrong information, potential for locking yourself out of the admin account if the e-mail ended up in junk, etc... Had to be resolved.

## How To Test This
* Set up the email settings for mailcatcher
* Log into the admin account
* Go to the Admin Panel
* Pick User Management
* Click on the name of a user account who isn't the Admin user.
* Click Actions -> Reset Password
* Check mailcatcher to make sure it was sending out to the user's email address not the admin's email address.

Please note: When you apply the code changes, please assure the cache directory has been cleared (or the template relating to the change has been deleted) as the template will have a cached id to it and will not be corrected, even with the code fix, until the file is deleted or rebuilt.

## Types of changes


### Final checklist
